aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/result/FederateToIdentityPoolResult.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.cognito.result
+
+import com.amplifyframework.statemachine.codegen.data.AWSCredentials
+
+/**
+ * Wraps the result of a federation to Identity Pool.
+ * @param credentials Returned credentials from successful federation
+ * @param identityId Identity Id of federation
+ */
+data class FederateToIdentityPoolResult(val credentials: AWSCredentials, val identityId: String)
```

Comment:
```suggestion
data class FederateToIdentityPoolResult(val identityId: String, val credentials: AWSCredentials)
```

For consistency with other credentials object with similar values.